client/components/Profile.jsx

```diff
@@ -6,63 +6,87 @@ import Pro from './Pro.jsx';
 
 export default class Profile extends Component {
     constructor() {
-    super();
+        super();
 
-    this.state = {
-        id: 22,
-        first_name: 'Joel',
-        last_name: 'Vandervort',
-        full_name: 'Joel Vandervort',
-        email: 'tbecker@example.net',
-        location: 'Fletachester, VA',
-        date_joined: '1979-02-20',
-        favorite_activities: ['Hiking', 'Hunting', 'Fishing'],
-        bio: 'Aut sequi dolore temporibus quia qui consequatur. Possimus non fugiat perspiciatis voluptas. Enim aut quia minus ipsam. Dolore corrupti et tenetur sit.',
-        photo_url: 'http://lorempixel.com/100/100/',
-        pro: false
-        // Could still add Reputation and followers
-    };
+        this.state = {
+            id: '',
+            first_name: '',
+            last_name: '',
+            full_name: '',
+            email: '',
+            location: '',
+            date_joined: '',
+            favorite_activities: [],
+            bio: '',
+            photo_url: '',
+            pro: false
+            // Could still add Reputation and followers
+        };
 
-    this.convertDate = this.convertDate.bind(this);
-  }
+        this.convertDate = this.convertDate.bind(this);
+    }
 
     convertDate() {
         const monthNames = ["January", "February", "March",
             "April", "May", "June", "July", "August", "September",
             "October", "November", "December"];
-        let date = new Date(this.state.date_joined +'Z');
+        let date = new Date(this.state.date_joined + 'Z');
 
         return monthNames[date.getMonth() - 1] + ' ' + date.getFullYear().toString();
     }
 
+    componentDidMount() {
+
+        fetch('http://localhost:3002/user/' + window.location.pathname.substring(1))
```

Comment:
hmm i just pull this down and your bundled version doesn't match the app.js in this commit seems the bundled version is pointing to an older saved versio you were working through.. did you do a webpack build beforehand? 
![image](https://user-images.githubusercontent.com/4239492/50048074-dbc8f800-0077-11e9-95b5-7fe7089789f3.png)

Reply:
also  i would recommend checking for validation... if window.location.pathname has a length and try parsing a number and if it fails, supply a default trail i.e. 1.